stripe/src/main/java/com/stripe/android/view/CardFormView.kt

```diff
@@ -0,0 +1,450 @@
+package com.stripe.android.view
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.Color
+import android.text.Editable
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.LayoutInflater
+import android.view.inputmethod.EditorInfo
+import android.widget.LinearLayout
+import androidx.annotation.VisibleForTesting
+import androidx.core.content.ContextCompat
+import androidx.core.content.withStyledAttributes
+import androidx.core.view.isVisible
+import androidx.core.view.updateLayoutParams
+import androidx.core.widget.doAfterTextChanged
+import com.stripe.android.R
+import com.stripe.android.databinding.StripeCardFormViewBinding
+import com.stripe.android.databinding.StripeHorizontalDividerBinding
+import com.stripe.android.databinding.StripeVerticalDividerBinding
+import com.stripe.android.model.Address
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.CardParams
+import com.stripe.android.view.CardFormView.Style
+import com.stripe.android.view.CardValidCallback.Fields
+
+/**
+ * A view to collect credit card information and provide [CardParams] for API invocation.
```

Comment:
Can we add a comment about how the view takes the customer's country and adjusts the postal code view based on their selection?